.github/workflows/build-test-osx-arm64.yml

```diff
@@ -27,7 +27,9 @@ jobs:
           key: ${{ runner.os }}-${{ runner.arch }}-opam-deps-4.14.0-${{hashFiles('semgrep.opam')}}
           path: ~/.opam
       - name: Install dependencies
-        run: ./scripts/osx-setup-for-release.sh "4.14.0"
+        run: |
+          ./scripts/osx-setup-opam-for-release.sh "4.14.0"
+          ./scripts/osx-setup-brew-for-release.sh
```

Comment:
```suggestion
          ./scripts/osx-setup-deps-for-release.sh
```

Reply:
Calling this post-opam per our discussion.